test/integration/controller/core/suite_test.go

```diff
@@ -43,7 +43,7 @@ var (
 	ctx         context.Context
 	fwk         *framework.Framework
 	crdPath     = filepath.Join("..", "..", "..", "..", "config", "components", "crd", "bases")
-	webhookPath = filepath.Join("..", "..", "..", "..", "config", "components", "webhook")
+	webhookPath = filepath.Join("..", "..", "..", "..", "config", "components", "webhook", "manifests.yaml")
```

Comment:
is this still necessary? since you moved the patches to the kustomize.yaml file.